### DISCUSSION TITLE
Fitting and running problem

### DISCUSSION THREAD

**Discussion by MohammadReza0GH** [phoebe-project/phoebe2#870] [Questions]
in Estimator lc geometry What could cause this incorrect fit?
![wrong](https://github.com/phoebe-project/phoebe2/assets/167889059/84c3bfc8-bdd1-4fc4-a833-e248483767dd)

and when i run OPTIMIZER after a few iters, solver is running but it does not progress no matter how long I wait. What could be the reason for this ??? i use pycharm.

**Comment by kecnry** [comment]
`lc_geometry` simply attempts to measure geometric properties of the phased light curve and proposes values for certain parameters.  It is designed and works best when eclipses are present in the light curve.  It is still up to you to decide whether to accept or reject the proposed values (so if you don't think they are reasonable, simply do not adopt the proposed changes).

As for the optimizer - what parameters are you including and which optimizer are you using?  I would generally suggest that you get a closer fit manually before switching to the optimizer - if the estimators do not help, then you can always set values manually to get to a model that is both physically reasonable and also decently represents the underlying data.  Generally manually intervention can be faster than relying on an optimizer (especially if you are unable to use a faster alternate backend).
